packages/node/src/integrations/http.ts

```diff
@@ -81,7 +81,7 @@ function createHandlerWrapper(
       let span: Span;
       if (tracingEnabled) {
         span = getCurrentHub().startSpan({
-          description: `${typeof options === 'string' || !options.method ? 'GET' : options.method}|${requestUrl}`,
+          description: `${typeof options === 'string' || !options.method ? 'GET' : options.method} ${requestUrl}`,
```

Comment:
Why is this changing?

Reply:
Because we dropped the `|` everywhere and just added a space.
Spotted this while going over the code.